packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -180,6 +180,15 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
     );
   }
 
+  protected okToAddMore(): boolean {
+    const { items } = this.state;
+    const { itemLimit } = this.props;
+    if (typeof itemLimit !== 'undefined' && items.length === itemLimit) {
```

Comment:
Also, you can further shorten by not even comparing undefined and just returning the length comparison. There really isn't a need to check itemLimit undefined unless you expect items.length to also potentially be undefined.

```tsx
return (
  items.length === itemLimit
);
```